library/tomee

```diff
@@ -55,5 +54,17 @@ Directory: 8-jre-8.0.0-M2-plus
 Tags: 8-jre-8.0.0-M2-microprofile
 Directory: 8-jre-8.0.0-M2-microprofile
 
-Tags: 8-jre-8.0.0-M2-webprofile, latest
+Tags: 8-jre-8.0.0-M2-webprofile
 Directory: 8-jre-8.0.0-M2-webprofile
```

Comment:
the 8.0.0-M2-webprofile directory does not exist anymore, please, update it to M3.